pipelines/upgrade/07-server_to_final.yaml

```diff
@@ -19,4 +19,6 @@
       scenario_os: "{{ pipeline_os }}"
       scenario_version: "{{ forklift_upgrade_version_final }}"
     - foreman_server_repositories
+    - update_os_packages
```

Comment:
I think this implies we end up running OS updates twice. It's confusing to me that our `foreman_installer` role has upgrade support but this is implemented only in a pipeline.

Reply:
I can put this in the foreman_installer role. That did not occur to me previously.